blog/_posts/2018-11-18-RGSoCTeamsGOTOBerlin.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Please remove this entire file from the pull request, as it's not relevant to the pull request, and also the two images associated with it!